### ISSUE TITLE
Receive Error "Request Exception: The Operation has Timed Out" with spinning hourglass when creating new campaign

### ISSUE THREAD

**Issue by corbinahenderson** [Bouncyrock/TaleSpire-Alpha-Public-Issue-Tracker#164]
**Describe the bug**
When trying to create my first campaign, after submitting the campaign name, I just see a spinning hourglass followed by an error message of Request Exception: The Operation has Timed Out".

**To Reproduce**
1. Open program
2. Click "Begin"
3. Click "Create New"
4. Name campaign and select "Submit"
5. ERROR- Request Exception: The Operation has Timed Out"
![Talespire Error](https://user-images.githubusercontent.com/63483721/79032899-f8941380-7b77-11ea-9f50-ea57a046ed59.PNG)

**Additional context**
I'm kind of new to Steam and PC gaming. I'm honestly not sure whether or not this is just a problem on my end.

**Comment by PleaseHLPMe** [comment]
I am having the same problem, I was curious whether or not it was a problem with the servers or maybe my internet connection specifically. Would you consider your connection weak or slow?

**Comment by corbinahenderson** [comment]
No, i have decent speed. I'm at around 50 Mbps on Upload and Download
speeds.
